cloudflare/resource_cloudflare_zone_settings_override_test.go

```diff
@@ -0,0 +1,131 @@
+package cloudflare
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccCloudFlareZone_Basic(t *testing.T) {
+	zoneName := os.Getenv("CLOUDFLARE_DOMAIN")
+	name := "cloudflare_zone_settings_override.test"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckCloudFlareZoneSettingsOverrideConfigEmpty(zoneName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet(
+						name, "settings.0.brotli"),
+					resource.TestCheckResourceAttrSet(
+						name, "settings.0.challenge_ttl"),
+					resource.TestCheckResourceAttrSet(
+						name, "settings.0.security_level"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccCloudFlareZone_Overrides(t *testing.T) {
```

Comment:
I can't verify this in my zones right now, everything is passing for me. I did notice in the TestAccCloudFlareZone_Basic there were some assertions that could fail based on the state of the underlying account which I have fixed ( the idea in this test case is to prove that the settings are unchanged from the initial settings). It doesn't look like any assertions in TestAccCloudFlareZone_Overrides should depend on the initial state of the zone. What error messages are you getting?

Reply:
@catsby Found some errors running tests when using a free instead of enterprise plan as I was before. Hopefully this will now be working for you in #41